ipaserver/install/server/__init__.py

```diff
@@ -335,6 +335,14 @@ def dirsrv_config_file(self, value):
     def __init__(self, **kwargs):
         super(ServerInstallInterface, self).__init__(**kwargs)
 
+        if self.domain_level == constants.DOMAIN_LEVEL_0:
+            if (self.no_pkinit or self.pkinit_cert_files is not None or
```

Comment:
If we allow `--no-pkinit` to be specified on DL0 the user might think that not-using it may change something for them IMO.